### PR TITLE
Added section with example for organizing tests

### PR DESCRIPTION
**Pull request by fireproofsocks**

### CODE REVIEW

File: docs/guide/testrunner/gettingstarted.md [hunk 1]
```diff
@@ -65,6 +65,17 @@ integration tests by calling:
...
+
+Since automation testing can be relatively time-intensive, it's important to plan how to
+organize your tests.  It is possible to organize your tests by creating multiple configuration files:
+each file can define a different list of "specs" directories.
+
+Additionally, Mocha's "grep" option is exposed to the `--mochaOpts` option, so you can use it to perform pattern matching
+on your test descriptions:
+
+```sh
+wdio --mochaOpts.grep "search term here"
```
Comment (christian-bromann) [inline docs/guide/testrunner/gettingstarted.md]: the path to the config file is missing here

**Comment by christian-bromann** [comment]
One comment, other than that LGTM

**Comment by fireproofsocks** [comment]
Can this be merged now?  I'm not sure what your comment refers to, to be honest.

**Comment by christian-bromann** [comment]
@fireproofsocks the wdio command expects a config file as second parameter

**Comment by fireproofsocks** [comment]
Optional path to wdio.conf.js now included in example.

**Comment by christian-bromann** [comment]
:+1:

**Pull request merged**